src/uikit.cs

```diff
@@ -328,25 +356,47 @@ partial interface NSFileProviderExtension {
 	    [Export ("documentStorageURL")]
 	    NSUrl DocumentStorageUrl { get; }
 	
-	    [Export ("URLForItemWithPersistentIdentifier:")]
-	    NSUrl GetUrlForItem (string persistentIdentifier);
+		[Deprecated (PlatformName.iOS, 13, 0)] // Undocumented replacement
+		[Export ("URLForItemWithPersistentIdentifier:")]
+		NSUrl GetUrlForItem (string persistentIdentifier);
 	
-	    [Export ("persistentIdentifierForItemAtURL:")]
-	    string GetPersistentIdentifier (NSUrl itemUrl);
+		[Deprecated (PlatformName.iOS, 13, 0)] // Undocumented replacement
+		[Export ("persistentIdentifierForItemAtURL:")]
+		string GetPersistentIdentifier (NSUrl itemUrl);
 	
-	    [Export ("providePlaceholderAtURL:completionHandler:")]
-	    [Async]
-	    void ProvidePlaceholderAtUrl (NSUrl url, [NullAllowed] Action<NSError> completionHandler);
+		[Deprecated (PlatformName.iOS, 13, 0)] // Undocumented replacement
+		[Export ("providePlaceholderAtURL:completionHandler:")]
+		[Async]
+		void ProvidePlaceholderAtUrl (NSUrl url, [NullAllowed] Action<NSError> completionHandler);
 	
-	    [Export ("startProvidingItemAtURL:completionHandler:")]
-	    [Async]
-	    void StartProvidingItemAtUrl (NSUrl url, [NullAllowed] Action<NSError> completionHandler);
+		[Deprecated (PlatformName.iOS, 13, 0)] // Undocumented replacement
+		[Export ("startProvidingItemAtURL:completionHandler:")]
+		[Async]
+		void StartProvidingItemAtUrl (NSUrl url, [NullAllowed] Action<NSError> completionHandler);
 	
-	    [Export ("itemChangedAtURL:")]
-	    void ItemChangedAtUrl (NSUrl url);
+		[Deprecated (PlatformName.iOS, 13, 0)] // Undocumented replacement
+		[Export ("itemChangedAtURL:")]
+		void ItemChangedAtUrl (NSUrl url);
 	
-	    [Export ("stopProvidingItemAtURL:")]
-	    void StopProvidingItemAtUrl (NSUrl url);
+		[Deprecated (PlatformName.iOS, 13, 0)] // Undocumented replacement
+		[Export ("stopProvidingItemAtURL:")]
+		void StopProvidingItemAtUrl (NSUrl url);
+
+		// TODO: Enable with new iOS 13 FileProvider bindings.
```

Comment:
better file an issue (or add it to the xcode11 checklist) or we might forget about it...

Reply:
Xtro never forgets! This is not removed from Xtro todo files, do you still want an issue for this? I just created the bindings because I needed the name in one of the Deprecated attributes